src/components/IconFrame.tsx

```diff
@@ -39,8 +39,8 @@ function typeToHoverBG(theme: typeof styledTheme): Record<Type, string> {
 
 function typeToSelectedBG(theme: typeof styledTheme): Record<Type, string> {
   return {
-    secondary: undefined,
-    tertiary: undefined,
+    secondary: theme.colors['fill-zero-selected'],
+    tertiary: theme.colors['fill-zero-selected'],
```

Comment:
I think these new colors will need versions for fill levels 1-3 as well. The `action-input-` colors just used a semi-transparent color, so it would be visible on any background level, but `fill-zero-hover` will look weird when the button is on a `fill-one`,`fill-two`, `fill-three` background.

Reply:
I added a new `onFillLevel` option to the story so we can see how it looks.

Reply:
You're right. Let's talk about it during frontend weekly.